niceml/mlcomponents/predictionhandlers/objdetpredictionhandler.py

```diff
@@ -37,7 +37,6 @@ def __init__(  # pylint: disable=too-many-arguments
         detection_idx_col: str = DETECTION_INDEX_COLUMN_NAME,
         apply_sigmoid: bool = True,
     ):
-
```

Comment:
Please add a docstring. I'm surprised that ruff doesn't mention the causes of the disabled pylint errors